src/index.tsx

```diff
@@ -479,6 +481,40 @@ const KEYS = {
   BACKSPACE: "Backspace"
 };
 
+const SHAPES = [
+  {
+    label: "Rectange",
+    value: "rectangle"
+  },
+  {
+    label: "Ellipse",
+    value: "ellipse"
+  },
+  {
+    label: "Arrow",
+    value: "arrow"
+  },
+  {
+    label: "Text",
+    value: "text"
+  },
+  {
+    label: "Selection",
+    value: "selection"
+  }
+];
+
+const shapesShortcutKeys = SHAPES.map(shape => shape.label[0].toLowerCase());
```

Comment:
Could you add an explicit shortcutKey? I feel like we'll likely want to change those in the future and would be nice to be explicit.

Reply:
If we want to make the shortcut key configurable (and not just the first letter), then we would need to also remove the UI indication (underlining of the first letter). I feel that if/when we add an explicit shortcutKey, we will forget to change the UI accordingly. Right now the functionality is 'coupled' with the UI. 

I don't feel really strong about it, WDYT?

Reply:
Let's ship it like this right now, if the need arises it's easy to change!